plugins/scaleflux/sfx-nvme.c

```diff
@@ -1453,20 +1454,12 @@ static int nvme_dump_evtlog(struct nvme_dev *dev, __u32 namespace_id, __u32 stor
 	printf("\nDump-evtlog: Success\n");
 
 	if (parse) {
-		nvme_free_huge(pevent_log_info, huge);
-		pevent_log_info = nvme_alloc_huge(log_len, &huge);
```

Comment:
Wait how come we can remove this new allocation? The original allocation was with `single_len`, and it seems like we now need a buffer of length `log_len`.

Reply:
Right, was confused. Fixed.